backend/scripts/seed_data.py

```diff
@@ -251,6 +252,18 @@
         ci_link="http://example17",
         test_plan="com.canonical.certification::sru-server",
     ),
+    StartCharmTestExecutionRequest(
+        family=FamilyName.CHARM,
+        name="postgresql-k8s",
+        version="123",
+        revision=123,
+        track="14",
+        arch="arm64",
+        execution_stage="candidate",
+        environment="juju=3.5 ubuntu=22.04 cloud=k8s",
```

Comment:
As mentioned in the meeting, the environment should be eventually revised to accept multiple values.